src/main/java/com/manardenza/dao/ReservationDaoImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.manardenza.dao;
+
+import com.manardenza.entity.Hotel;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.manardenza.entity.Hotel'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)